### DISCUSSION TITLE
mspider/test/ocp

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2931] [Announcements]
# mspider/test/ocp

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/test/ocp.html

**Comment by hanxiaop** [comment]
ocp安装流程在新版本有一些优化，可以参考 istio.io 的内容提及一下

**Comment by Fan-Lin** [comment]
OCP、Openshift 还是 OCP Openshift，术语如无法统一，最好遵循特定逻辑。
